### ISSUE TITLE
Fix Request: AvoidDuplicateAssignmentsInCases false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#561]
```
switch (statusCode) {
    case "A39" :
         errorCode = ErrorCodes.NOT_ALLOWED.getErrorCode();
         break;
    case "A45" :
         errorCode = ErrorCodes.NOT_AVAILABLE.getErrorCode();
         break;
    }
```
